### ISSUE TITLE
[BUG] VoiceOver Rotor on modal includes content from entire website

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3866]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When opening a modal, the VoiceOver rotor (headings, links, form-tools etc.) contain elements the entire website. Ie. Including the site-menu and the page from which the modal was opened.

## Describe how to reproduce the bug
In the Kirby Stackblitz press the "Open modal" button. Open VoiceOver rotor (VO+U) and see that it contains a lot of items, not related to the modal itself.

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
The VoiceOver rotor should only contain items from the modal itself, in order to give the non-seeing user a chance to navigate within the modal, and not by accident to choose eg. a link or a button from the page behind the modal.

## Add any screenshots
https://github.com/user-attachments/assets/bb25af76-5f40-48f2-858e-56fa1f3efdf3

## Please complete the following information:
- OS: macOS
- Browser: Chrome
- Browser version: n/a

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
